### DISCUSSION TITLE
This browser doesn't provide means to serialize canvas to an image

### DISCUSSION THREAD

**Discussion by caredev** [fabricjs/fabric.js#6969] [General]
http://fabricjs.com/kitchensink
It does not rasterize the canvas to image. Is there any way to fix it?
I get this alert This browser doesn't provide means to serialize canvas to an image

**Comment by asturur** [comment]
'This browser' what browser?

**Reply by caredev** [reply]
chrome / brave sorry.

**Comment by asturur** [comment]
Can you take a screenshot of the error? never seen that
